utils/upload_ansible_roles_to_galaxy.py

```diff
@@ -219,9 +266,27 @@ def parse_args():
         help="Path to directory containing the generated roles. Most "
         "likely this is going to be ./build/roles",
         dest="build_roles_dir")
+    parser.add_argument(
+        "--organization", "-o", default=ORGANIZATION_NAME,
+        help="Name of the Github organization")
+    parser.add_argument(
+        "--profile", "-p", default=[], nargs="*", action="append",
+        help="What profiles to upload, if not specified, upload them all.")
```

Comment:
This may give the impression that it will upload all profiles, while only the ones in allowed list will be uploaded.